examples/ColorPicker.main.jsx

```diff
@@ -20,9 +23,20 @@ export default () => {
   };
 
   return (
-    <ColorPicker selectedColor={activeColor} onChangeComplete={onColorChanged}>
-      <ColorBuilder />
-      <ColorInputPanel defaultColorFormat={'hsl'} />
-    </ColorPicker>
+    <Popover
+      content={
+        <ColorPicker
+          selectedColor={activeColor}
+          onChangeComplete={onColorChanged}
+        >
+          <ColorBuilder />
+          <ColorInputPanel defaultColorFormat={'hsl'} />
+        </ColorPicker>
+      }
+    >
+      <IconButton>
```

Comment:
This `IconButton` should have a label, like the on in the advanced popover example. The example doesn't meet accessibility requirements without it.